[FW][FIX]stock_voucher_ux: print vouchers button

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#616

mav-adhoc committed:
[FIX]stock_voucher_ux: print vouchers button

X-original-commit: 7708d920fb50df2d4098217a6c751ad4185f1d6f

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/617).

fw-bot-adhoc commented:
@mav-adhoc @jcadhoc cherrypicking of pull request ingadhoc/stock#616 failed.

stdout:
```
CONFLICT (modify/delete): stock_voucher_ux/views/stock_picking_views.xml deleted in HEAD and modified in 1f18f647 ([FIX]stock_voucher_ux: print vouchers button). Version 1f18f647 ([FIX]stock_voucher_ux: print vouchers button) of stock_voucher_ux/views/stock_picking_views.xml left in tree.
CONFLICT (modify/delete): stock_voucher_ux/__manifest__.py deleted in HEAD and modified in 1f18f647 ([FIX]stock_voucher_ux: print vouchers button). Version 1f18f647 ([FIX]stock_voucher_ux: print vouchers button) of stock_voucher_ux/__manifest__.py left in tree.

```

stderr:
```
15:45:50.893944 git.c:444               trace: built-in: git cherry-pick 1f18f6476f593488ca2e51fb45cd2c617907a68b
error: could not apply 1f18f647... [FIX]stock_voucher_ux: print vouchers button
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
